definitions/npm/express_v4.x.x/flow_v0.25.x-/express_v4.x.x.js

```diff
@@ -12,7 +12,7 @@ declare module 'express' {
   }
   declare class Request extends http$IncomingMessage mixins RequestResponseBase {
     baseUrl: string;
-    body: mixed;
+    body: any;
```

Comment:
@jeffmo shouldn't everything in javascript be an `Object` ? what is the point of `any` in this case?